src/components/navbar/navbar.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { FaBars } from 'react-icons/fa';
+import { FaShoppingCart } from 'react-icons/fa';
+import { BiDotsVertical } from 'react-icons/bi'
+import logo from '../../assets/images/Renome.svg';
+import './navbar.scss';
+
+const Navbar = () => {
+    const [showLinks, setShowLinks] = useState(true);
+    const [showCart, setShowCart] = useState(true)
+
+
+    return (
+        <nav>
```

Comment:
`<nav>` is not necessary to use. You have already wrapped everything with `<div className='nav__center'>`